src/ui/veles_mainwindow.cc

```diff
@@ -281,6 +284,23 @@ void VelesMainWindow::updateConnectionStatus(
       == client::NetworkClient::ConnectionStatus::Connected) {
     database_dock_widget_->widget()->setEnabled(true);
     open_act_->setEnabled(true);
+
+    if (!have_ever_been_connected_) {
```

Comment:
do we realy need this flag here, why do not use files_to_upload_once_connected_.empty() ?